CheckerboardKarel.java

```diff
@@ -1,16 +1,107 @@
-/*
- * File: CheckerboardKarel.java
- * ----------------------------
- * When you finish writing it, the CheckerboardKarel class should draw
- * a checkerboard using beepers, as described in Assignment 1.  You
- * should make sure that your program works for all of the sample
- * worlds supplied in the starter folder.
- */
-
-import stanford.karel.*;
-
-public class CheckerboardKarel extends SuperKarel {
-
-	// You fill in this part
-
-}
+/*
+ * File: CheckerboardKarel.java
+ * ----------------------------
+ * When you finish writing it, the CheckerboardKarel class should draw
+ * a checkerboard using beepers, as described in Assignment 1.  You
+ * should make sure that your program works for all of the sample
+ * worlds supplied in the starter folder.
+ */
+
+import stanford.karel.*;
+
+public class CheckerboardKarel extends SuperKarel {

```

Comment:
Your code assumes that you are working in a world that is 8x8 but the assignment expects your solution to work with different types of world...because of that your solution is not good enough.